### PR TITLE
feat(ui): Show version information on about page

### PR DESCRIPTION
**Pull request by lamppu**
Näyttökuva (92)

### CODE REVIEW

**Comment by sschuberth** [comment]
> Resolves #608

Please put this (with a trailing dot) into the commit message, not in the PR discussion.

File: ui/src/routes/_layout/about/index.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  return (
+    <>
+      <Card className='mx-auto w-full max-w-4xl'>
+        <CardHeader>
+          <h2 className='text-3xl font-bold tracking-tight'>About</h2>
+        </CardHeader>
+        <Separator />
+        <CardContent className='pt-6'>
+          <div>
+            <h3 className='font-semibold'>Server and tool versions:</h3>
```
Comment (sschuberth) [inline ui/src/routes/_layout/about/index.tsx:42]: As the data can contain arbitrary names and versions, I'd more generally just say "Version information:".

File: ui/src/routes/_layout/about/index.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+import { Card, CardContent, CardHeader } from '@/components/ui/card';
+import { Separator } from '@/components/ui/separator';
+
+export const About = () => {
+  const { data: versionData } = useSuspenseQuery({
+    queryKey: [useVersionsServiceGetVersions],
+    queryFn: () => VersionsService.getVersions(),
+  });
+  return (
+    <>
```
Comment (mmurto) [inline ui/src/routes/_layout/about/index.tsx:34]: No need for `<>` as there's only one direct child.

**Review by mmurto — CHANGES_REQUESTED** [review]

**Review by mmurto — APPROVED** [review]

**Comment by sschuberth** [comment]
@lamppu stupid question maybe, but how do I actually access the about screen? I don't see it anywhere...

**Comment by lamppu** [comment]
> @lamppu stupid question maybe, but how do I actually access the about screen? I would have expected it to be accessible from the profile icon, but it's not there for me:
> 
> image

Oh yeah sorry, I forgot to add the link. You can browse to /about for now (e.g. http://localhost:5173/about)

**Comment by sschuberth** [comment]
> Oh yeah sorry, I forgot to add the link.

I gave it a try to add it at https://github.com/eclipse-apoapsis/ort-server/pull/639.

**Pull request merged**